src/playground/App.js

```diff
@@ -401,6 +415,24 @@ const App = () => {
 								text={validationError.message}
 							/>
 						)}
+						{messages.filter(
+							message =>
+								!message.suggestions &&
+								(message.fix || options.rules[message.ruleId]),
+						).length > 1 && (
```

Comment:
```suggestion
						{filteredMessages.length > 1 && (
```
Do this calculation outside the render block.


```
const filteredMessages = messages.filter(
  message =>
    !message.suggestions &&
    (message.fix || options.rules[message.ruleId])
);
```